src/components/Editor/Editor.js

```diff
@@ -28,7 +32,12 @@ export default function Editor() {
       </div>
       <div className="form-control">
         <label>Alignment</label>
-        <div className="radio-group">
+        <div
+          className="radio-group"
+          onClick={(e) => {
+            setAlign(e.target.value);
+          }}
+        >
           <label>
             <input name="align" type="radio" value="left" />
```

Comment:
Excellent work Kamie!! If you wanted to add the default highlighted alignment, we can add checked={align === 'left'} in the input field of the radio button. We will want to do that for each input within the radio-group div and have align === 'center'/'right'. It looks like you are super comfortable passing down props and using state within React, great job! :)